src/DistributedLock.Azure/packages.lock.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": 1,
+  "version": 2,
```

Comment:
Interesting that the `version` changed from 1 to 2. Given that there is a new `type` `CentralTransitive`, I guess that makes sense.